unlock-js/src/__tests__/web3Service.ethers.test.js

```diff
@@ -76,4 +77,18 @@ describe('Web3Service', () => {
       expect(web3Service.provider).toBeInstanceOf(ethers.providers.Web3Provider)
     })
   })
+
+  describe('getAddressBalance', () => {
+    it('should return the balance of the address', async () => {
+      expect.assertions(1)
+      await nockBeforeEach()
+      const balance = '0xdeadbeef'
+      const inWei = utils.hexToNumberString(balance)
+      const address = '0x1df62f291b2e969fb0849d99d9ce41e2f137006e'
+      nock.getBalanceForAccountAndYieldBalance(address, '0xdeadbeef')
+
+      let addressBalance = await web3Service.ethers_getAddressBalance(address)
+      expect(addressBalance).toEqual(utils.fromWei(inWei, 'ether'))
```

Comment:
It might be a good idea to ensure that the righthand side of this is an integer for easier validation of the tests

Reply:
re-organized the test a bit to help with readability. The right-hand side is not in fact an integer, it's a bigNumber, which is a struct with a `_hex` field that is set to a hex string

Reply:
(which is what `ethers_getAddressBalance` returns)